capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -7,26 +7,51 @@ class BookSearchList extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookList: []
+      bookSearchList: []
     }
-    this.getData = this.getData.bind(this);
   }
 
-  getData() {
+  getData = () => {
     fetch(`/api/search?searchTerm=${this.props.searchQuery}`)
       .then(response => response.json())
-      .then(res => this.setState({ bookList: res }))
+      .then(res => this.setState({ bookSearchList: res }))
```

Comment:
nit: `res` here is actually the body/payload of the response I think, so calling it something like `books` would probably be more accurate.

Reply:
Fixed!